README.md

```diff
@@ -1,3 +1,88 @@
 Ansible Role: Filebeat
 =========
 
+Install Filebeat on CentOS servers.
+
+Requirements
+------------
+
+Written in Ansible 1.9.*
+
+Role Variables
+--------------
+
+Available variables are listed below, along with default values (see `defaults/main.yml`):
+
+### supervisord_config_dir
+
+Directory where supervisord reads configuration files.
+
+Default is `/etc/supervisor.d`.
+
+### filebeat_supervisor_enabled
```

Comment:
I added this config item to make the dependency optional. We already had it set up this way for other roles so I figured it's a nice touch.